content/en/blog/2023/sig-architecture-conformance-spotlight.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: blog
+title: "Spotlight on SIG Architecture: Conformance"
+slug: sig-architecture-conformance-spotlight-2023
+date: 2023-09-28
+author: "Frederico Muñoz (SAS Institute)"
+---
+
+_This is the first interview of a SIG Architecture Spotlight series
+that will cover the different subprojects. We start with the SIG
+Architecture: Conformance subproject_
+
+In this [SIG
+Architecture](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md)
+spotlight, we talked with [Riaan
+Kleinhans](https://github.com/Riaankl) (ii-Team), Lead for the
+[Conformance
+sub-project](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md#conformance-definition-1).
+
+## About SIG Architecture and the Conformance subproject
+
+**Frederico (FSM)**: Hello Riaan, and welcome! For starters, tell us a
+bit about yourself, your role and how you got involved in Kubernetes.
+
+**Riaan Kleinhans (RK)**: Hi! My name is Riaan Kleinhans and I live in
+South Africa. I am the Project manager for the [ii-Team](ii.nz) in New
+Zealand. When I joined ii the plan was to move to New Zealand in April
+2020 and then Covid happened. Fortunately, being a flexible and
+dynamic team we were able to make it work remotely and in very
+different time zones.
+
+The ii team have been tasked with managing the Kubernetes Conformance
+testing technical debt and writing tests to clear the technical
+debt. I stepped into the role of project manager to be the link
+between monitoring, test writing and the community. Through that work
+I had the privilege of meeting [Dan Kohn](https://github.com/dankohn)
```

Comment:
Should we link to a memorial page/post just to make sure folks understand Dan is no longer with us?

Reply:
Added a link to the memorial page, and also "the late" to make it clearer.